php-checks/src/test/resources/checks/security/CORSPolicyCheck.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace core;
+
+header("Access-Control-Allow-Origin: *"); // Noncompliant
+header("Access-Control-Allow-Origin:* "); // Noncompliant
+Header("access-control-allow-origin:     *   "); // Noncompliant
+header(code:42, replace:true, header:"Access-Control-Allow-Origin: *"); // Noncompliant
+
+header("Access-Control-Allow-Origin: " + $_SERVER['HTTP_ORIGIN']); // Compliant - FN
+header("Access-Control-Allow-Origin: *foo*");
+header("Access-Control-Allow-Origin", "*");
+header("Other-Property: *"); // Compliant
+header(); // Compliant
+
+class A {
+    function foo() {
+        header("Access-Control-Allow-Origin: *"); // Noncompliant
+        header("Access-Control-Allow-Origin: foo"); // Compliant
+    }
+}
```

Comment:
I don't really see the point of such test cases.

Reply:
True.